wellcomecollection/catalogue-pipeline
# make sure path is a string

## What does this change?

`obj.get` returns an Option, which caused the collectionPath to be serialised as `Some("A/B")` 
Using flatmap to get the value of the Option 

## How to test

The `pathProcessor` logs the input paths and selectors. 
We can see with the change that they are values, not Some(value)
From
`Input paths (1-6): Some("A/B"), Some("A/B/C/D"), Some("E/F/G"), Some("F/G/H"), Some("Y/Z"), Some("Z")`
to
`Input paths (1-6): "A/B", "A/B/C/D", "E/F/G", "F/G/H", "Y/Z", "Z"`

## How can we measure success?

The batcher service and lambda output the same values 

## Have we considered potential risks?

The batcher lambda output is not subscribed yet
